lib/build/loader.js

```diff
@@ -0,0 +1,82 @@
+"use strict"
+
+const path = require('path');
+const LoaderPlugin = require('./loader-plugin').LoaderPlugin;
+
+exports.generateLoaderConfig = function generateLoaderConfig(platform, bundler) {
```

Comment:
@EisenbergEffect, I was struggling with the naming. I like that more than what I had. But is it clear enough that people who call it know what they're going to get? I guess that could just be documented somewhere.